lib/msal-browser/src/config/Configuration.ts

```diff
@@ -122,7 +123,11 @@ export function buildConfiguration({ auth: userInputAuth, cache: userInputCache,
         postLogoutRedirectUri: "",
         navigateToLoginRequestUrl: true,
         clientCapabilities: [],
-        protocolMode: ProtocolMode.AAD
+        protocolMode: ProtocolMode.AAD,
+        azureCloudOptions: {
+            azureCloudInstance: AzureCloudInstance.None,
```

Comment:
How does the default work if the developer provides `azureCloudOptions` but only provides the `tenant` property? Do we still give `azureCloudInstance` a default value somewhere?

Reply:
Good point, I don't think so, we default to "none". It should still hold good, as we check for `azureCloudInstance` value instead. Let me add a test though, to check the validity of this.

Reply:
Added the test.